### PR TITLE
[OneBranch] Adding UAP back into the pipeline

### PR DESCRIPTION
**Pull request by gladjohn**
We are now able to install the Windows 10 SDK (v 17763), I will follow that up with a[ OneBranch PR](https://identitydivision.visualstudio.com/IDDP/_git/MSAL.NET-OneBranch/pullrequest/9068). This fix will add back UAP to OneBranch again.

### CODE REVIEW

File: tests/devapps/WAM/UWPWam/MainPage.xaml.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ private IPublicClientApplication CreatePublicClient()
             var pca = PublicClientApplicationBuilder.Create(s_clientID)
                 .WithAuthority(s_authority)
                 .WithBroker(chkUseBroker.IsChecked.Value)
-                //.WithWindowsBrokerOptions(new WindowsBrokerOptions() { HeaderText = "aaa" })
```
Comment (trwalke) [inline tests/devapps/WAM/UWPWam/MainPage.xaml.cs:51]: dead code?

**Review by trwalke — APPROVED** [review]

File: src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <TargetFrameworkNetStandard>netstandard2.0</TargetFrameworkNetStandard>
     <TargetFrameworkNet6>net6.0</TargetFrameworkNet6>
     <TargetFrameworkNet6Win>net6.0-windows</TargetFrameworkNet6Win>
+    <TargetFrameworkUap>uap10.0.17763</TargetFrameworkUap>
```
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj:9]: Can you please add this to the "legacy" property group (line 13). We should not build this by default

**Review by bgavrilMS — APPROVED** [review]

**Review by pmaytak — APPROVED** [review]

File: src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj (hunk 2 above)
Comment (gladjohn) [reply src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj]: updated

File: tests/devapps/WAM/UWPWam/MainPage.xaml.cs (hunk 1 above)
Comment (trwalke) [reply tests/devapps/WAM/UWPWam/MainPage.xaml.cs:51]: Oh, nvm, my mistake

**Review by trwalke — APPROVED** [review]

**Pull request merged**